caller.py

```diff
@@ -0,0 +1 @@
+from pydantic import Required
```

Comment:
This file shouldn't be here.

Reply:
Will be moved into tests after initial feedback :)

Reply:
Best to start tests and not add this to the PR.